File: 1. C, C++/1. Bronze/1259.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main() {
    // Variables
    string a, b;

    while(true) {
        // Take Input
        cin >> a;
        b = a;

        // Loop Break Condition
        if ((a.length() == 1) && (a[0] == '0')) break;

        // Reverse
        reverse(b.begin(), b.end());

        // Print Result
        if (equal(a.begin(), a.end(), b.begin())) cout << "yes" << endl;
        else cout << "no" << endl;
    }
}

File: 2. Java/1. Bronze/1259.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        // Variables
        Scanner sc = new Scanner(System.in);
        String a, b;

        while(true) {
            // Take Input
            a = sc.next();
            b = a;

            // Loop Break Condition
            if ((a.length() == 1) && (a.charAt(0) == '0')) break;

            // Reverse
            StringBuffer str = new StringBuffer(a);
            b = str.reverse().toString();

            // Print Result
            if (a.equals(b)) System.out.println("yes");
            else System.out.println("no");
        }
    }
}